controller/src/main/java/io/pravega/controller/server/rpc/auth/PasswordAuthHandler.java

```diff
@@ -79,17 +79,19 @@ public String getHandlerName() {
     public Principal authenticate(String token) throws AuthException {
         String[] parts = parseToken(token);
         String userName = parts[0];
-        String password = parts[1];
+        char[] password = parts[1].toCharArray();
```

Comment:
Should there be a null check here, in case `parts[1]` is null? Or, maybe some sort of an input validation to ensure input is valid?

Reply:
https://github.com/pravega/pravega/blob/7f76baa4674f1e572cc0571bb22f505ae0583076/controller/src/main/java/io/pravega/controller/server/rpc/auth/PasswordAuthHandler.java#L122-L126
No since `io.pravega.controller.server.rpc.auth.PasswordAuthHandler#parseToken` already validates if the `parts.length ==2` and `java.lang.String#split(java.lang.String, int)` ensures no null string is returned.

Reply:
Sounds good.